public/javascripts/SVValidate/css/svv-panorama.css

```diff
@@ -68,3 +72,7 @@
     font-size: 12px;
     z-index: 1;
 }
+
+.label-description-box.amsterdam {
+    visibility: visible;
+}
```

Comment:
Can you make sure to keep newlines at the ends of files, as per [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide).